cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -49,7 +49,7 @@ var addRemoveBackoff = wait.Backoff{
 // ClusterInterrogator is an interface to get etcd cluster related information
 type ClusterInterrogator interface {
 	ClusterAvailable() (bool, error)
-	GetClusterStatus() (map[string]*clientv3.StatusResponse, error)
+	GetClusterStatus(noResponseMap bool) (map[string]*clientv3.StatusResponse, error)
```

Comment:
I don't think we should add a flag to the interface, because the only consumer that uses the return value is inside the etcd.go file.
what about removing the return value and renaming into CheckClusterHealth